{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -168,3 +169,11 @@
 
 # This is needed to get a CRSF token in /admin
 ANON_ALWAYS = True
+
+USE_X_FORWARDED_HOST = config('USE_X_FORWARDED_HOST', default=True, cast=bool)
+SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=not DEBUG, cast=bool)
+SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', cast=int)
```

Comment:
Having it set to '0' means it does nothing at all. Sugardough is about good defaults that keep you safe without having to change much. 
The correct value is actually `31536000` (which is 1 year). That means you're telling browsers that, for the next year, never accept any HTTP responses on this domain.